package.json

```diff
@@ -11,8 +11,9 @@
     "db-seed": "NODE_ENV=testing sequelize db:seed:all",
     "test": "NODE_ENV=testing npm run dropTables && NODE_ENV=testing npm run db-migrate && npm run db-seed && NODE_ENV=testing nyc --reporter=text --reporter=html --require @babel/polyfill --require @babel/register mocha ./src/tests/*  --timeout 200000 --exit",
     "coverage": "nyc npm test  && npm run coveralls-coverage",
-    "dev-start": "nodemon --exec babel-node ./src/index.js",
+    "dev-start": "npx sequelize-cli db:seed:all && nodemon --exec babel-node ./src/index.js",
     "create-db": "npx sequelize-cli model:generate --name users --attributes firstName:string,lastName:string,email:string,password:string,isVerified:string,signupType:string && sequelize db:migrate",
+    "create-db-triprequestcities": "npx sequelize-cli model:generate --name tripRequestCities --attributes tripRequestId:integer cityId:integer",
```

Comment:
To reduce the complexity of our package.json, I recommend you save script models in your  `.env` file.